decidim-elections/app/packs/src/decidim/errors.js

```diff
@@ -0,0 +1,24 @@
+export function reportingErrors(fn) {
+    return function () {
+      try {
+        return fn.apply(this);
+      } catch (e) {
+        alert(e);
```

Comment:
Using `alert()` is not accessible so could you please use the custom modals that we are using for other alerts / confirmations as well.

Here is for example the confirm modal implementation:
https://github.com/decidim/decidim/blob/8014ebc2416e745bc2d482c7a0280c9f0e37d4b7/decidim-core/app/packs/src/decidim/confirm.js#L12